.circleci/config.yml

```diff
@@ -53,6 +55,10 @@ jobs:
             python examples/int100.py
             python examples/inputs.py
             python examples/matmul.py
+      - run:
```

Comment:
We need to use a tool like `make` or similar to wrap up all these different commands.